src/modules/commander/Arming/ArmAuthorization/ArmAuthorization.cpp

```diff
@@ -43,12 +43,17 @@
 #include <uORB/topics/vehicle_command.h>
 #include <uORB/topics/vehicle_command_ack.h>
 
+using namespace time_literals;
+
 static orb_advert_t *mavlink_log_pub;
 static int command_ack_sub = -1;
 
-static param_t param_arm_parameters;
-
 static hrt_abstime auth_timeout;
+static hrt_abstime auth_req_time;
+
+static hrt_abstime _param_com_arm_auth_timout;
+static enum arm_auth_methods _param_com_arm_auth_method;
```

Comment:
It would be nice to restructure this without all the static variables at some point.

Reply:
I note that and will see if i can do this someday